packages/search/style.css

```diff
@@ -9,10 +9,8 @@
   --mobile-margin-negative: -24px;
   --nav-height: 64px;
 
-  /* Allow focus ring to be fully visible on Safari */
-  padding-top: 3px;
```

Comment:
This offset threw me off, as it wrecks the top alignment of the search bar with the adjacent `docs-sidenav`.

I made some tweaks to `docs-page` styles -  previously the `#inner` element had `overflow: auto` set, which was causing the focus ring to be cut off. By tweaking the `flex`-related styles, (ie setting `width: 0` so the content area grows due to `flex` rather than shrinking), `overflow: auto` could be removed along with this offset.